### PR TITLE
Add html, css and javascript files

### PR DESCRIPTION
**Pull request by Bellagirl-maker**
Peer feedback on the to-do-list Project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+  <div class="site-wrapper">
+    <section class="todo-section">
+      <h2>To Do list </h2>
+      <ul class="todo-box">
+        <!-- Generate List from Javascript  -->
+      </ul>
+    </section>
+  </div>
```
Comment (massazoum) [inline index.html:19]: - Kindly link you JavaScript file to your main HTML file by this way <script src=" **main jf file** " ></script>

File: index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html> 
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todo List</title>
+  <script src="https://kit.fontawesome.com/a0d7ff3ed5.js" crossorigin="anonymous"></script>
```
Comment (massazoum) [inline index.html]: - kindly put your style file in this head part for that your work become well with  UI design

**Review by massazoum — COMMENTED** [review]
Hi @Bellagirl-maker  👋👋, 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f69656d6d596a3332456945655a59766a70712f67697068792e676966


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅

- [x] Gitflow practice has been used👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Bellagirl-maker) [reply index.html:19]: @massazoum I have made this change as requested. Thanks 👍

File: index.html (hunk 2 above)
Comment (Bellagirl-maker) [reply index.html]: @massazoum I have included the css file in the head tag of my HTML file. Thanks 👍

**Review by massazoum — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @massazoum, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions
- Kindly Write a meaningful description for your pull request 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**